src/ui/common/src/components/workflows/storageSelector.tsx

```diff
@@ -0,0 +1,59 @@
+import Box from '@mui/material/Box';
+import FormControl from '@mui/material/FormControl';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { RootState } from '../../stores/store';
+import { StorageType, StorageTypeNames } from '../../utils/storage';
+
+export const StorageSelector: React.FC = () => {
+  const workflow = useSelector((state: RootState) => state.workflowReducer);
+  const dag = workflow.selectedDag;
+  let selected = 'file';
+  if (dag) {
+    selected = dag.storage_config.type;
+  }
+
+  const getMenuItems = () => {
+    return (
+      Object.values(StorageType) as Array<
+        typeof StorageType[keyof typeof StorageType]
+      >
+    ).map((r, _) => {
+      return (
+        <MenuItem
+          value={r}
+          key={r}
+          sx={{ backgroundColor: selected ? 'blueTint' : null }}
+        >
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography sx={{ ml: 1 }}>{StorageTypeNames[r]}</Typography>
+          </Box>
+        </MenuItem>
+      );
+    });
+  };
+
+  const menuItems = getMenuItems();
+
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      Storage Type:{' '}
+      <FormControl disabled sx={{ m: 1, minWidth: 120 }} size="small">
+        <Select
+          sx={{ maxHeight: 50 }}
```

Comment:
Can you use a unit of 8 here like 48 or 56 please?

Reply:
I used the same structure & styling as the VersionSelector (except I added `m: 1` for this component.) I've updated both to be units of 8 (maxHeight: 48).

It doesn't seem to have much of a change to the UI:
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/30596854/194022330-547b8850-07c5-4793-918f-cf7bcea91e63.png">